clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -110,10 +108,15 @@
       new ConcurrentHashMap<String, Set<ClientDescriptor>>();
 
   private final ServerStoreCompatibility storeCompatibility = new ServerStoreCompatibility();
+  private final EhcacheEntityResponseFactory responseFactory;
 
-  EhcacheActiveEntity(ServiceRegistry services, byte[] config) {
+  EhcacheActiveEntity(ServiceRegistry services, byte[] config, EhcacheEntityResponseFactory responseFactory) {
```

Comment:
Again, I don't get passing in a response factory.  I seriously doubt we're going to ever have a different set of message objects for this -- a different _codec_ maybe but not messages.